x/tokenomics/token_logic_module/types.go

```diff
@@ -60,28 +66,35 @@ func (tlm TokenLogicModuleId) EnumIndex() int {
 // actors if their stake falls below a certain threshold.
 type TokenLogicModule interface {
 	GetId() TokenLogicModuleId
```

Comment:
Should ID be Name?

Reply:
I chose "id" because the enum type uses `int`s; therefore, I didn't see "name" as an accurate description. They _have_ names, but are not names in and of themselves. Do you see it differently?